ambari-server/src/main/java/org/apache/ambari/server/utils/ScheduledExecutorCompletionService.java

```diff
@@ -35,6 +35,7 @@
       super(task, null);
       this.task = task;
     }
+    @Override
     protected void done() { queue.add(task); }
```

Comment:
can we also reformat the line to have proper formatting?   (not one-liner)

thanks

Reply:
It's done.